src/components/ExampleButton/ExampleButton.tsx

```diff
@@ -7,16 +7,18 @@ export interface ExampleButtonProps {
 
 export const ExampleButton = ({ variant, label }: ExampleButtonProps) => {
   const variants = {
-    primary: "bg-redMain text-backgroundMain hover:bg-[#7A0003] active:bg-[#520103]",
-    inversion: "bg-backgroundMain text-redMain hover:bg-backgroundB active:bg-[#e8e3da]",
+    primary:
+      "bg-redMain text-backgroundMain hover:bg-[#7A0003] active:bg-[#520103]",
+    inversion:
+      "bg-backgroundMain text-redMain hover:bg-backgroundB active:bg-[#e8e3da]",
     secondary: "bg-greenLight text-black hover:bg-greenC active:bg-greenB",
   };
 
   return (
     <button
-    type="button"
-      className={clsx(
-        "font-sourceSans button-outline h-fit w-[160px] rounded px-8 py-3 text-lg",
+      type="button"
+      className={cn(
+        "button-outline h-fit w-[160px] rounded px-8 py-3 font-sourceSans text-lg",
```

Comment:
Is there any specific reason we should use cn instead of clsx? I ask out of sheer curiosity. Also there is no need to update this component. It is to be removed as soon as other components are introduced into the project and was only meant to serve as a template for the file structure :)

Reply:
`cn` uses _tailwind-merge_ which automatically resolves conflicts by keeping the last applicable class. It's particularly useful when combining multiple class strings that might have overlapping utilities. But in this component it's unnecessary, you're right 😄

Reply:
Thanks, didn't know that 😉  in that case I will update the button component, I used clsx there too.